backend/internal/models/goals.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+
+
+type Goal struct {
+	DbModel
+	UserId   			uint          	`json:"user_id"`
+	GameId   			uint          	`json:"game_id"` 
+	//Game     			Game          	`gorm:"game"` TODO: When seeding is finished we can officially test this
+	TeamId   			uint          	`json:"team_id"` 
+	//Team     			Team			`gorm:"team"` TODO: When seeding is finished we can officially test this
+	Duration 			uint			`json:"duration"` // Do we potentially want to change this to time.Duration
+	Period	    		uint			`json:"period"`
+	Assist1Id			uint   	   		`json:"assist1_id"`
+	Assist2Id			uint   	   		`json:"assist2_id"`
+	PlayerDifferential	int		   		`json:"playerdifferential"`
```

Comment:
Is there a reason why this is an int while the others are uint?

Reply:
note to other pull request